source/docs/magento2/add-new-attribute.md

```diff
@@ -0,0 +1,187 @@
+Learn more about Front-Commerce's folder structure [here](/docs/essentials/extend-the-theme.html).
+
+```bash
+mkdir mymodule
+cd mymodule
+```
+
+In this example, the files we are going to override are located at `node_modules/front-commerce/src/web/theme/modules/ProductView/Synthesis/`, so will create in our `mymodule` folder the following directories. More details [here](/docs/essentials/extend-the-graphql-schema.html).
+
+```bash
+mkdir -p server/modules/acme
+```
+
+(Note that the `acme` name has been chosen arbitrarily.)
+
+## Add the module to Front-Commerce
+
+Now we will add the module we just made to Front-Commerce. Edit the `.front-commerce.js` file (root folder) as shown below to do so.
+
+```diff
+module.exports = {
+  name: "Front Commerce Skeleton",
+  url: "http://localhost:4000",
+- modules: ["./src"],
++ modules: ["./mymodule", "./src"],
+  serverModules: [
+    { name: "FrontCommerce", path: "server/modules/front-commerce" },
+-   { name: "Magento2", path: "server/modules/magento2" }
++   { name: "Magento2", path: "server/modules/magento2" },
++   { name: "Acme", path: "./mymodule/server/modules/acme" }
+  ]
+};
+```
+
+### Expand the Product definition
+
+Create a file named `schema.gql` and type in the following code. It means that we are expanding the definition of a product: it can now be rated.
+
+```js
+// mymodule/server/modules/acme/schema.gql
+extend type Product {
+  rate: Float
+}
+```
+
+### Implement the resolver
+
+We must now define what Front-Commerce should fetch when `rate` is requested in a GraphQL query; in other words we must write the code that will **resolve** the queries for the rates.
+
+Create a file named `resolvers.js` and type in the following code. It implements the resolver.
+
+```js
+// mymodule/server/modules/acme/resolvers.js
+export default {
+  Product: {
+    rate: parent => {
+      return parseFloat(parent.rate);
+    }
+  }
+};
+```
+
+### Declare the module
+
+If we want our code to be taken into account when the module is loaded, we must reference it.
+
+In the file named `index.js`, type in the following code. It references the schema and the resolver from earlier.
+
+```js
+// mymodule/server/modules/acme/index.js
+import typeDefs from "./schema.gql";
+import resolvers from "./resolvers";
+
+export default {
+  namespace: "Acme",
+  typeDefs: typeDefs,
+  resolvers: resolvers
+};
+```
+
+## Discover the playground
+
+By typing `yourhostname/playground` or `yourhostname/graphiql` (in our case `localhost:4000/playground`) in your browser address bar, you can access a GraphQL playground with a nice GUI to test your queries. For instance, type the following code in the left pane and press `Ctrl` + `Enter` (or click the arrow button)
```

Comment:
Don't forget to mention the fact that one need to restart their application before accessing to the updated schema in the playground.